### ISSUE TITLE
Console not longer working like it should 

### ISSUE THREAD

**Issue by NicolaiVdS** [pieceofsummer/Hangfire.Console#94]
I just tested this extension with the last version of hangfire and it trys to display the 'console' but not it should 
![preview](https://i.imgur.com/hoofWOL.png)

also the text doesnt live update i have to refresh the page to get new lines also it dosnt scroll (dono if that was a feature)

**Comment by pieceofsummer** [comment]
I believe it is browser caching issue. Force reloading page or clearing cache should fix this.

**Comment by NicolaiVdS** [comment]
that is correct sorry for the false issue, also is there a feature coming for making the scrolling possible ?
